pkgs/development/tools/rust/dioxus-cli/default.nix

```diff
@@ -32,10 +34,11 @@ rustPlatform.buildRustPackage rec {
     "--skip=server::web::proxy::test::add_proxy_trailing_slash"
   ];
 
-  doInstallCheck = true;
-  installCheckPhase = ''
-    $out/bin/dx --version | grep "dioxus ${version}"
-  '';
+  passthru.tests.version = testers.testVersion {
+    package = dioxus-cli;
+    command = "${meta.mainProgram} --version";
+    inherit version;
+  };
```

Comment:
Not sure if this is correct, namely wether `passthru.tests.version` is run as intended and a full replacement of the install check.

Reply:
Seems to work fine.
Tested by changing 'version' to '0.5.1'.

```
$ nix build .#dioxus-cli.passthru.tests.version
error: builder for '/nix/store/lp1c11ggf98yiwai7v9pxpb9gwn326g8-dioxus-cli-0.4.1-test-version.drv' failed with exit code 1;
       last 3 log lines:
       > Version string '0.5.1' not found!
       > The output was:
       > dioxus 0.4.1
       For full logs, run 'nix log /nix/store/lp1c11ggf98yiwai7v9pxpb9gwn326g8-dioxus-cli-0.4.1-test-version.drv'.
```